popup.js

```diff
@@ -1,52 +1,70 @@
 
     /* Creates a form that allows for the user to input text to later be used in a note. */
-    plusNote.onclick = function writeNote() {
-        noteForm.classList.toggle("show");
-        noteForm.reset();
 
-        if (!plusNote.classList.contains("bi-x-circle")) {
-            plusNote.classList.add("bi-x-circle");
-            plusNote.classList.remove("bi-plus-circle");
-        }
-        else {
-            plusNote.classList.remove("bi-x-circle");
-            plusNote.classList.add("bi-plus-circle");
+    global.plusNote.onclick = function writeNote() {
+        global.noteForm.classList.toggle("show");
+        global.noteForm.reset();
+
+        if (!global.plusNote.classList.contains("bi-x-circle")) {
+            global.plusNote.classList.add("bi-x-circle");
+            global.plusNote.classList.remove("bi-plus-circle");
+        } else {
+            global.plusNote.classList.remove("bi-x-circle");
+            global.plusNote.classList.add("bi-plus-circle");
         }
     };
 
-    /* Creates an instance of a note using the text submission from noteForm. Prevents the deafult action of form submission in order to grab contents of the textarea. */
-    submitNote.onclick = function addNote(ev) {
-        ev.preventDefault(); //stop the form submitting
+    /* Creates an instance of a note using the text submission from noteForm. Prevents the 
+     * default action of form submission in order to grab contents of the textarea. */
 
-        let note = {
-            text: document.getElementById("text").value
-        }
-        text = note.text;
+    global.submitNote.onclick = function addNote(ev) {
+
+        //Stop the form submitting.
 
-        noteForm.reset();
+        ev.preventDefault();
+
+        let note = {
+            text: document.getElementById("text").value,
+        };
+        global.text = note.text;
 
-        noteForm.classList.remove("show");
+        global.noteForm.reset();
+        global.noteForm.classList.remove("show");
 
-        var newNote = document.createElement('div');
-        newNote.setAttribute("class", "note");
+        //Adds in the text content that is in the note. CSS takes these elements into 
+        //account. Text variable holds the text from the above function that copies an
+        //instance of a note from the HTML template.
+        
+        var temp = document.querySelector("#note");
+        var clone = temp.content.cloneNode(true);
+        var div = clone.querySelector(".text");
+        div.textContent = global.text;
 
-        /* Adds in the inner html that is basically the note. CSS takes these elements into account. Text variable holds the text from the above function that creates an instance of a note */
-        newNote.innerHTML = "<div class='text'>" + text + "</div><div class='edits-container'><i id='edit' class='bi bi-pencil'></i><i id='delete' class='bi bi-trash'></i></div>";
+        //Appends child (clone of note template) to the top.
```

Comment:
Should have a space after the `//`

Reply:
I just added the space to this comment and all others of the like that require a space after the comment indication! Thank you!